presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -416,4 +418,28 @@ public PinotConfig setNonAggregateLimitForBrokerQueries(int nonAggregateLimitFor
         this.nonAggregateLimitForBrokerQueries = nonAggregateLimitForBrokerQueries;
         return this;
     }
+
+    public boolean isInferDateTypeInSchema()
+    {
+        return inferDateTypeInSchema;
+    }
+
+    @Config("pinot.infer-date-type-in-schema")
```

Comment:
shall we add documentation in:
./presto-docs/src/main/sphinx/connector/pinot.rst

Reply:
Added docs